testsuite/utils.py

```diff
@@ -26,3 +29,26 @@ def _whoami():
     # pylint: disable=broad-except
     except Exception:
         return str(os.getuid())
+
+
+def cert_builder(cfssl: CFSSLClient, chain: dict, hosts: Union[str, Collection[str]] = None,
+                 parent: Certificate = None) -> Dict[str, Certificate]:
+    """Create certificates according to their CertInfo"""
```

Comment:
This deserves more documentation. Try to describe recursion flow (maybe in few steps)